src/Microsoft.ServiceFabric.Client/IClusterClient.cs

```diff
@@ -711,5 +711,12 @@ public partial interface IClusterClient
         Task<AadMetadataObject> GetAadMetadataAsync(
             long? serverTimeout = 60,
             CancellationToken cancellationToken = default(CancellationToken));
+
+
+        /// <summary>
+        /// Gets the connection string for the image store on the current cluster
+        /// </summary>
+        /// <returns></returns>
+        Task<string> GetImageStoreConnectionString();
```

Comment:
This interface is auto-generated from swagger specification, which generates all interfaces and classes as partial to allow manual extension later. Please add this extra method in a partial interface in other file somethign like IClusterClient_